.github/workflows/test.yml

```diff
@@ -4,35 +4,45 @@ on:
   workflow_dispatch:
     inputs:
       environment:
-        type: string
+        type: environment
         description: Name of the environment to use
+        default: "oee-dev"
+        options:
+          oee-dev: oee-dev Environment
         required: true
       release_body:
         type: string
         description: GitHub Release Message
-        default: "GitHub Release"
+        default: "GitHub Manual Release"
 
   push:
-    branches: [ '*' ]
+    branches: [ main ]
     environment:
-      name: Nam
+      name: oee-dev
 
 env:
-  C8Y_BASEURL: ${{ secrets.C8Y_BASEURL }}
   C8Y_PASSWORD: ${{ secrets.C8Y_PASSWORD }}
-  C8Y_TENANT: ${{ secrets.C8Y_TENANT }}
-  C8Y_USER: ${{ secrets.C8Y_USER }}
+  C8Y_BASEURL: ${{ vars.C8Y_BASEURL }}
+  C8Y_TENANT: ${{ vars.C8Y_TENANT }}
+  C8Y_USER: ${{ vars.C8Y_USER }}
 
 jobs:
   run-tests:
     runs-on: ubuntu-latest
-    environment: Nam
+    environment: ${{ github.event.deployment.environment || github.event.inputs.environment }}
```

Comment:
Again, if we're only running manually, don't need to choose between environments.